proto/api.go

```diff
@@ -308,14 +307,139 @@ func (sr *ReverseScanResponse) Verify(req Request) error {
 	return nil
 }
 
+// GetInner returns the Request contained in the `oneof`.
+func (ru BatchRequest_RequestUnion) GetInner() Request {
+	switch t := ru.Request.(type) {
```

Comment:
Why do you have to type switch to the implementation. Can't you do something like:

```
  switch t := ru.Request.(type) {
    case Request:
      return t
  }
```